tests/common/plugins/loganalyzer/__init__.py

```diff
@@ -34,6 +34,14 @@ def loganalyzer(duthost, request):
     logging.info("Load config and analyze log")
     # Read existed common regular expressions located with legacy loganalyzer module
     loganalyzer.load_common_config()
+    # sflowmgrd is configed in Monit, but not included in current master branch image
+    # So a workaround is added for master branch. Once the issue is adressed, we should
+    # rm this patch
+    if "master" in duthost.os_version:
```

Comment:
os_version contains 'master' is a very specific check that only works for images built by public Jenkins and/or no tag on the latest commit. not sure if we should keep it here or ignore the event in general until the fix is in?

Reply:
> os_version contains 'master' is a very specific check that only works for images built by public Jenkins and/or no tag on the latest commit. not sure if we should keep it here or ignore the event in general until the fix is in?

I add the check for **master** intendedly because I expect this patch only work for a few days, as PR  https://github.com/Azure/sonic-buildimage/pull/5153 will address it. Will this issue exist in next 201911 build?

Reply:
If this ignore will only be needed for a few days, maybe it is better to update the common ignore file in jenkins just before running the tests?

Reply:
Oh. That's quite a creative idea. I will apply this easy patch on Jenkins now. Thanks Xin.